packages/Confirmation/src/Confirmation.js

```diff
@@ -98,32 +98,28 @@ const Confirmation = props => {
 
   const I18n = useI18n();
 
-  // note: In future could support a node instead of just function
-  const triggerComponent = children ? children(triggerProps) : null;
-
-  return (
-    <Popover isOpen={isConfirmOpen} onClose={handleCloseConfirm} {...moreProps}>
-      {triggerComponent && <Popover.Trigger>{renderTrigger(triggerComponent)}</Popover.Trigger>}
-      <Popover.Content id={confirmId.current}>
-        <Popover.Card>
-          <div css={confirmStyles}>
-            {heading && (
-              <Heading displayLevel={5} level={2}>
-                {heading}
-              </Heading>
-            )}
-            {body && <div css={confirmBodyStyles}>{body}</div>}
-            <div css={confirmFooterStyles}>
-              <Button
-                isPending={isPending}
-                isSemantic={false}
-                ref={confirmButtonRef}
-                kind={confirmButtonType}
-                size={buttonSize}
-                onClick={handleOnConfirm}
-              >
-                {confirmLabel}
-              </Button>
+  const popoverContent = (
+    <Popover.Content id={confirmId.current}>
+      <Popover.Card>
+        <div css={confirmStyles}>
+          {heading && (
+            <Heading displayLevel={5} level={2}>
+              {heading}
+            </Heading>
+          )}
+          {body && <div css={confirmBodyStyles}>{body}</div>}
+          <div css={confirmFooterStyles}>
+            <Button
+              isPending={isPending}
+              isSemantic={false}
+              ref={confirmButtonRef}
+              kind={confirmButtonType}
+              size={buttonSize}
+              onClick={handleOnConfirm}
+            >
+              {confirmLabel}
+            </Button>
+            {!isPending ? (
```

Comment:
I am hiding the cancel button if the state of the confirm button is now Pending. If we do not do this then we get wrapping issues 
![image](https://user-images.githubusercontent.com/4040102/70580096-73680600-1b67-11ea-9eca-109bb63f3a14.png).
Perhaps another approach would be to to re-render then entire popover content tree (which would ensure the popover renders content width correctly) or to call an imperative api callback on the popover which re-calculates the content width, but i had issues with both approaches. Open to suggestions for either if we are adamant we don't want the cancel button to disappear, Personally I think this approach is fine.

Reply:
I dont think removing the cancel button is the ideal way to go (1: for a11y, 2: if some day we support actually cancelling the async request)

Reply:
Cancelling the async could still be done, just change the `isPending` prop back to true and the cancel button would show. I'll look into re-rendering of the popup content as I agree it's not ideal..